src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -522,9 +523,13 @@ public boolean isMethod() {
     public boolean isObjectConstructor() {
         return testAllFlags(IS_OBJECT_CONSTRUCTOR);
     }
+    /** Query whether this member is a constructor. */
+    boolean isConstructor(String name) {
```

Comment:
`findConstructor` only finds `<init>`.  To get `<vnew>` method handle, it just uses `Lookup::findStatic` as it's a static method.   This `isConstructor` method would cause confusion.  I can clean this up as a follow-up.

Reply:
Thank you